### PR TITLE
Add FastMLST v0.0.4

### PR DESCRIPTION
**Pull request by EnzoAndree**
In the previous pull request, I had made an error not adding the build.sh file and a name error in the meta.yaml file.

Now everything should be fine.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by EnzoAndree** [comment]
@BiocondaBot please add label

File: recipes/fastmlst/build.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py install --single-version-externally-managed --record=/tmp/record.txt
```
Comment (bwlang) [inline recipes/fastmlst/build.sh:3]: Why is this needed? I see you tried to embed it in meta.yaml earler, but I don't see why that did not work. I'm alos not sure why --record and --single-version-externally-managed are necessary

Comment (EnzoAndree) [reply recipes/fastmlst/build.sh:3]: Hello! Thank you for your comment @bwlang 

I added `--single-version-externally-managed --record=/tmp/record.txt ` because without that line the local test generated an error `RuntimeError: Setuptools downloading is disabled in conda build. Be sure to add all dependencies in the meta.yaml  url=https://pypi.org/simple/biopython/`

I realized that by removing the build.sh file and adding `as` to the meta.yaml the local test works.

Do you think that is better to add all the installation logic to the meta.yaml?
If you think so, I could add a new update to the recipe with that change

**Comment by bwlang** [comment]
will there be any confusion with https://github.com/B-UMMI/fast-mlst ?

File: recipes/fastmlst/build.sh (hunk 1 above)
Comment (bwlang) [reply recipes/fastmlst/build.sh:3]: Maybe it's best to list the dependencies in the meta.yaml and avoid the need for this build.sh script?
looks like it 's just 'tqdm', 'pandas', 'biopython' right?

Comment (EnzoAndree) [reply recipes/fastmlst/build.sh:3]: I listed the dependencies ´tqdm, pandas, biopython´ in requirements/run in the meta.yaml. That looks fine?

I'm going to update the recipe without build.sh

**Comment by EnzoAndree** [comment]
> will there be any confusion with https://github.com/B-UMMI/fast-mlst ?

I did not expect that there would be a program with such a similar name. However, it is not in bioconda.
Will it be necessary to modify the name? maybe `pymlst` should be used.

**Comment by bwlang** [comment]
> > will there be any confusion with https://github.com/B-UMMI/fast-mlst ?
> 
> I did not expect that there would be a program with such a similar name. However, it is not in bioconda.
> Will it be necessary to modify the name? maybe `pymlst` should be used.

I don't think this is a bioconda issue... just thought you'd want to know...

File: recipes/fastmlst/build.sh (hunk 1 above)
Comment (bwlang) [reply recipes/fastmlst/build.sh:3]: sounds like it should work nicely - i'll look for the update, (unless someone else gets to it first ;)

**Comment by EnzoAndree** [comment]
> > > will there be any confusion with https://github.com/B-UMMI/fast-mlst ?
> > 
> > 
> > I did not expect that there would be a program with such a similar name. However, it is not in bioconda.
> > Will it be necessary to modify the name? maybe `pymlst` should be used.
> 
> I don't think this is a bioconda issue... just thought you'd want to know...

Thanks for your help to this newbie.
I hope everything is in order now.

**Comment by bwlang** [comment]
I'm not sure I really understand why the build script is needed at all... but I see that other packages follow the same pattern.
@bioconda/core  ok to merge?

**Review by bgruening — APPROVED** [review]

**Comment by bwlang** [comment]
@BiocondaBot  please merge
